### ISSUE TITLE
[steamshovel] log viewer setting not saved (Trac #1587)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1579]
In window->configuration, there is a setting to "automatically open log viewer"

This doesn't get saved in the session.

Migrated from https://code.icecube.wisc.edu/ticket/1587
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:26", 
    "description": "In window->configuration, there is a setting to \"automatically open log viewer\"\n\nThis doesn't get saved in the session.", 
    "reporter": "david.schultz", 
    "cc": "cweaver", 
    "resolution": "fixed", 
    "_ts": "1550067086520250", 
    "component": "combo core", 
    "summary": "[steamshovel] log viewer setting not saved", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2016-03-16T02:29:29", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160316T15:27:52: 
* hdembinski commented:

> In changeset 143307:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="143307"
> remember whether to autostart the logviewer, closes #1579
> ```


* hdembinski changed status from "new" to "closed"
* hdembinski changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:26: 
* olivas commented:

> Milestone offline V16-03-00 deleted


* olivas changed milestone from "offline V16-03-00" to ""
